Makefile

```
@@ -423,7 +423,7 @@ build-mixin: check-mixin-jb
 		./tools/check-rules.sh "$(MIXIN_OUT_PATH)$$suffix/rules.yaml" 20 ; \
 		cd "$(MIXIN_OUT_PATH)$$suffix/.." && zip -q -r "mimir-mixin$$suffix.zip" $$(basename "$(MIXIN_OUT_PATH)$$suffix"); \
 		cd -; \
-		echo "The mixin has been compiled to $(MIXIN_OUT_PATH)$$suffix and archived to $$(realpath --relative-to=$$(pwd) $(MIXIN_OUT_PATH)$$suffix/../mimir-mixin$$suffix.zip)"; \
+		echo "The mixin has been compiled to $(MIXIN_OUT_PATH)$$suffix and archived to $$(realpath $(MIXIN_OUT_PATH)$$suffix/../mimir-mixin$$suffix.zip)"; \
```

How about simplifying and defining the `OPERATIONS_PATH` variable, representing the parent directory of mimir mixins? Then you avoid the contortion of using `realpath`, and `operations/mimir-mixin-baremetal.zip` will be printed instead of `/go/src/github.com/grafana/mimir/operations/mimir-mixin-baremetal.zip` when building inside a container:
```suggestion
		echo "The mixin has been compiled to $(MIXIN_OUT_PATH)$$suffix and archived to $(OPERATIONS_PATH)/mimir-mixin$$suffix.zip"; \
```

---

I think printing the full path is fine for now - if it bothers us, we can always change it later.

---

The full path is also a bit misleading it should be said, since it refers to the path within the cointainer, and not where the file gets written in the host filesystem.